python/setup.py

```diff
@@ -249,7 +250,11 @@ def build(build_python, build_java):
 
     version_info = bazel_invoke(subprocess.check_output, ["--version"])
     bazel_version_str = version_info.rstrip().decode("utf-8").split(" ", 1)[1]
-    bazel_version = tuple(map(int, bazel_version_str.split(".")))
+    bazel_version_split = bazel_version_str.split(".")
+    bazel_version_digits = [
+        "".join(takewhile(str.isdigit, s)) for s in bazel_version_split
```

Comment:
Nice, thanks! Which Bazel version did this break on?

Reply:
```
$ bazel --version
bazel 3.4.1- (@non-git)
```
aka the one I installed from the Arch repos. Parsed wrong because of the extra bit at the end.

Reply:
Ahh, thanks!